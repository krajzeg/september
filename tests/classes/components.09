Person := class (Person) {
	field firstName = "John"
	field lastName = "Kowalski"

	method (fullName) {
		return firstName + " " + lastName
	}
}

person := Person()
print("If things went smoothly, this should say John Kowalski:", person.fullName())
