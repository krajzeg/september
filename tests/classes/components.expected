If things went smoothly, this should say John Kowalski: John Kowalski
And this, Maggie Smith: Maggie Smith
